resources/assets/sass/layout/_header.scss

```diff
@@ -56,6 +56,9 @@ header.navbar-default {
     font-size: 9px;
     border: 1px solid #fff;
     border-radius: 5px;
+    overflow: inherit;
```

Comment:
hmm, nie wiem jaki to ma związek. na prawdę poprawa styli w tym miejscu odbija się na zawijaniu nicków userów? :) sprawdzę to :)

Reply:
Opisałem związek na forum :)